src/components/CameraButton.tsx

```diff
@@ -47,7 +47,13 @@ function CameraButton(props: CameraButtonProps) {
 
   return (
     <div className={classes.margin}>
-      <InputLabel error={props.error} disabled={staticPreview} htmlFor={props.id} className={classes.label}>
+      <InputLabel
+        required={props.required}
+        error={props.error}
+        disabled={staticPreview}
+        htmlFor={props.id}
+        className={classes.label}
```

Comment:
Is there a way to test this? The receipt wasn't required on the "Purchase Request" form. Should it have been?